src/components/tiles/Tile.tsx

```diff
@@ -80,16 +80,10 @@ const defaultStyles = createStyles('Tile', (theme) => ({
     height: 55,
     borderRadius: 10,
     backgroundColor: theme.colors.surface,
-    shadowColor: theme.colors.shadow,
-    shadowOpacity: 0.1,
-    shadowOffset: {
-      height: 2,
-      width: 0,
-    },
-    shadowRadius: 8,
-    marginHorizontal: theme.spacing.large,
-    marginBottom: theme.spacing.extraSmall,
-  },
+    marginHorizontal: theme.spacing.medium,
+    marginBottom: theme.spacing.medium,
+    ...shadow(2),
+  } as ViewStyle,
```

Comment:
Super cool `shadow` function from Paper that takes an elevation.   Have to look into enabling our Type system with it in a subsequent PR.